_posts/2018-06-13-Exploring-The-vRo-Plugin-Part1.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: Exploring the Nuage vRealize Orchestrator Plugin - Part 1
+author: Leonard Paquette
+callout_image: nuage-vro-header.jpg
+excerpt: If your Nuage VSP environment is integrated with VMware, then there's a good chance you are using vRealize Orchestrator (`vRO`) to automate many of your networking tasks.  The Nuage Networks VSP vRO Plugin operates within `vRO` to help you to manage your SDN by offering links to the Virtualized Services Directory (`VSD`) VSPK.  In Part 1 of this series, we provide screenshots from vRO to accompany steps in the plugin's installation guide and then walk through some examples of using the plugin to create, edit and remove a basic VSD object.
+
+---
+# Introduction
+If your Nuage VSP environment is integrated with VMware, then there's a good chance you are using vRealize Orchestrator (`vRO`) to automate many of your networking tasks.  The Nuage Networks VSP vRO Plugin operates within `vRO` to help you to manage your SDN by offering links to the Virtualized Services Directory (`VSD`) VSPK.  In Part 1 of this series, we provide screenshots from vRO to accompany steps in the plugin's installation guide and then walk through some examples of using the plugin to create, edit and remove a basic VSD object.
+
+# Product Overview
+VMware is both a leader and pioneer in platform virtualization and cloud computing.  The company offers many products and tools to their customers to manage these functions, from bare-metal hypervisors to automation and deployment tools.  The VMware vRealize Suite is a complete Cloud Management Platform for managing an enterprise's private and public cloud resources.
+
+The VMware vRealize Orchestrator (`vRO`) tool integrates with the vRealize Suite to automate network operations.  It offers an easy-to-use interface to combine VMware specific functions and third-party functions into `workflows` that can then be executed manually, scheduled for later execution, triggered from an external script, or run automatically after notification of an external network event.  With vRO, a user could assemble a set of tasks into workflows that interface with vCenter, Active Directory, PowerShell and other endpoints.  

```

Comment:
Can you make this `to automate IT operations`, it is more than network operations... Same for 'external network event', let's change that to `external event`.

Reply:
will change today